src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import javafx.application.Application;
+import javafx.stage.Stage;
+;
+
+/**
+ * Main file for the duke bot. The duke bot keeps track of all your task and allows you to mark as complete.
+ * Call list to see the list of task. Each task can be completed. When a task is completed, it will still appear
+ * when you call list this time, but will be deleted from history for storage purposes the next time it is ran.
+ * Use dukebot to mark down tasks today!
+ */
```

Comment:
Same comment as before! I thought throughout your documentation was great. Sorry for reposting but 2103 required review comments